app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def most_recent_posts(user_id)
+    User.find_by(id: user_id).posts.order(created_at: :desc).limit(3)
```

Comment:
- [OPTIONAL] Suggestion
```suggestion
    posts.order('created_at Desc').limit(3)
```